lib/cisco_node_utils/interface.rb

```diff
@@ -74,8 +74,6 @@ def access_vlan
 
     def access_vlan=(vlan)
       config_set('interface', 'access_vlan', @name, vlan)
-    rescue Cisco::CliError => e
-      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
```

Comment:
I saw your comment in the PR description but I don't quite understand why you're removing these completely? 
These were originally added because we had a hard time determining which interface was failing when it hit an error. Is the interface name preserved and still displayed somewhere when we hit a cli error?